ocs_ci/utility/baremetal.py

```diff
@@ -244,3 +246,110 @@ def get_nodes_ipmi_ctx(self, baremetal_machine):
                 )
                 node_ipmi_ctx.append(ipmi_ctx)
         return node_ipmi_ctx
+
+
+def update_uefi_boot_order(
+    first_option_string=None, ocp_node=None, host=None, ignore_error=False
+):
+    """
+    Update UEFI boot order (using efibootmgr) to set option containing first_option_string to first place.
+    Depending on the ocp_node and host parameters, it tries to access the node via oc debug command (if ocp_node
+    parameter is provided) or via ssh (as core user) if host (hostname or IP) is provided.
+    Parameter ocp_node have precedence before host parameter, which means that if both parameters are provided, it
+    will try to use only the oc debug access.
+
+    Args:
+        first_option_string (str): String identifying the boot option which should be set to first place
+        ocp_node (str): OCP Node name used to access the server via oc debug command
+        host (dict): configuration of the server to make ssh connection or None
+                the dict could contain following keys: host, user and private_key or password
```

Comment:
I think you have one more indention here

Reply:
fixed